umap/tests/integration/test_edit_datalayer.py

```diff
@@ -178,3 +179,10 @@ def test_can_restore_version(live_server, openmap, page, datalayer):
     page.once("dialog", lambda dialog: dialog.accept())
     page.get_by_role("button", name="Restore this version").last.click()
     expect(marker).to_have_class(re.compile(".*umap-ball-icon.*"))
+
+
+def test_can_edit_layer_on_ctrl_shift_click(live_server, openmap, page, datalayer):
```

Comment:
All the removed tests are covered by just this? Maybe it's missing the actual functional tests somewhere? 🤔

Reply:
:upside_down_face: